tests/unit/amo/components/TestFlagReviewMenu.js

```diff
@@ -65,30 +65,6 @@ describe(__filename, () => {
     expect(root.find(TooltipMenu)).toHaveProp('openerClass', openerClass);
   });
 
-  it('passes a custom className when flagged', () => {
```

Comment:
We are passing now a same class for all the classes so i dont think having these tests make sense. WDYT Bob?

Reply:
I agree.